experiments/current-topology-poc/main.go

```
@@ -0,0 +1,2964 @@
+	deadline := time.Now().Add(wait)
+	for {
+		if activeUnix(path, 200*time.Millisecond) {
+			if wait > 0 && time.Now().Before(deadline) {
+				time.Sleep(100 * time.Millisecond)
+				continue
+			}
+			return nil, fmt.Errorf("listener already active at %s", path)
+		}
+
+		_ = os.Remove(path)
+		ln, err := net.Listen("unix", path)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)